src/__tests__/Home.test.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import GlobalContextProvider from '../app/contexts';
+import App  from '../App';
+import store  from '../app/store';
+import { authSlice } from '../features/auth/authDuck';
+import * as firebase from '../app/firebase-configs';
+import * as selectorMonthlyBudget from '../features/monthly-budget/useSelectorForMonthlyBudgetStatus'
+
+const mockBasicRequestData = {
+  month: 6,
+  project: {
+      createdAt: "2021-07-18T23:31:00.842Z",
+      name: "Principal",
+      userUid: "QovcZZVRTrd9Fq3glumc734pLrz1",
+      uuid: "iRqmxVou7FVNfPTbZXOW"
+  },
+  user: { displayName: "Marizinha", email: "teste@teste.com", uid: 123123 },
+  year: 2021,
+}
+
+const mockIncomes = {
+  "total": {
+    toString() {
+      return "86.6"
+    },
+    isZero() {
+      return false
+    },
+    lessThan() {
+      return false
+    },
+    toFixed(...args){
+      return Number(this.toString()).toFixed(...args)
+    }
+  },
+  "totalIncomes": "19.52",
+  "totalExpenses": "23.92"
+}
+
+jest.mock('../app/useBasicRequestData', () => jest.fn(() => mockBasicRequestData))
+jest.spyOn(firebase, 'onIdTokenChanged').mockImplementation(() => jest.fn())
+
+jest.spyOn(selectorMonthlyBudget, 'getMonthlyCalcs').mockImplementation(() => (mockIncomes))
```

Comment:
youre excessively mocking, usually mocking a pure function like this is a bad smell of depending way too much on implementation details.

just run the application locally, fill it with some data* using the GUI, open your browser inspector and run `store.getState()`, this will show up fixture data, so you can use the fixtures to build your own `store`.

*it isnt any data, take a look on the trello card for the different kinds of data.